packages/extension/src/background/actionHandlers.ts

```diff
@@ -85,6 +85,13 @@ export const handleActionApproval = async (
       }
     }
 
+    case "REQUEST_TOKEN": {
```

Comment:
How about `REQUEST_ADD_TOKEN`? The one from Argent is a bit misleading IMO.

Reply:
Great idea, will fix